lib/fizzy/capi.cpp

```diff
@@ -127,8 +127,10 @@ inline fizzy::ValType unwrap(FizzyValueType value_type) noexcept
 inline FizzyFunctionType wrap(fizzy::span<const fizzy::ValType> input_types,
     fizzy::span<const fizzy::ValType> output_types) noexcept
 {
+    // NOLINTBEGIN(readability-container-data-pointer)
     return {(output_types.empty() ? FizzyValueTypeVoid : wrap(output_types[0])),
         (input_types.empty() ? nullptr : wrap(&input_types[0])), input_types.size()};
```

Comment:
clang-tidy dislikes `&input_types[0]`.

Reply:
Did you try using `data()` instead as suggested in https://clang.llvm.org/extra/clang-tidy/checks/readability/container-data-pointer.html ?

Reply:
I tried but got more errors, maybe didn't do it properly.

Reply:
Fixed.